infrastructure/main.tf

```
@@ -0,0 +1,17 @@
+provider "aws" {
+  region = var.region
+}
+
+data "aws_caller_identity" "current" {
+}
+
+data "terraform_remote_state" "idp_authorizer_state" {
```

"-state" in name is redundant

```suggestion
data "terraform_remote_state" "idp_authorizer" {
```